invenio_app_ils/records_relations/api.py

```diff
@@ -158,11 +158,20 @@ def _validate_relation_between_records(self, parent, child, relation_name):
             and parent["mode_of_issuance"] == "SERIAL"
         )
 
-        if not (is_series_doc or is_serial_mm):
+        child_has_mm = False
+        if relation_type.name == "multipart_monograph":
+            pcr = ParentChildRelation(relation_type)
+            relations = pcr.get_relations_by_child(child.pid)
+            if len(relations) > 0:
+                child_has_mm = True
+
+        if not (is_series_doc or is_serial_mm) or child_has_mm:
```

Comment:
if we check if child has mm on the beginning, we can avoid checking the rest of the conditions, but also provide more useful error message (f.e. `Record {pid} has already a multipart monograph`)
but it could also stay like this.
Please add a test case